src/ray/common/ray_config_def.h

```diff
@@ -84,6 +84,14 @@ RAY_CONFIG(float, memory_usage_threshold_fraction, 0.9)
 /// Monitor is disabled when this value is 0.
 RAY_CONFIG(uint64_t, memory_monitor_interval_ms, 0)
 
+/// The interval between checking whether task failure entry
+/// should be garbage collected. Disabled when value is 0.
+RAY_CONFIG(uint64_t, task_failure_entry_gc_period_ms, 0)
+
+/// The TTL for when the task failure entry is considered
+/// eligble for garbage colletion.
+RAY_CONFIG(uint64_t, task_failure_entry_ttl_ms, 15 * 60 * 1000)
```

Comment:
Can we only add this setting instead of both? Even better would be if we could reuse an existing setting, but I'm not sure if there's one that would fit.

Reply:
Bump on this comment.

Reply:
can you clarify which setting to keep and which one to remove?

Reply:
Er, I don't feel strongly about it, but semantically seems like just specifying a TTL would make sense. (I mainly just want to try to minimize how many settings we add to this config)

Reply:
so do we hard code the period ? I checked but haven't found an existing setting that makes sense to piggy back on (if we do we might as well run the logic together with an existing periodic fn)

Reply:
How about hard-coding the period to TTL * 2?